src/main/java/org/threadly/litesockets/UDPServer.java

```diff
@@ -145,5 +164,20 @@ public void run() {
     }
     
   }
+  
+  /**
+   * 
+   * @author lwahlmeier
+   *
+   */
+  private static class WriteDataPair {
```

Comment:
yeah, I have used that else where, seemed like there was not much need here since I dont need anything else that class does.